src/utils/getHyperLinkFromPath.ts

```diff
@@ -1,16 +1,14 @@
-/**
- * generates the clickable link to a specific folder
- * This will recieve the path of a specific folder
- * Gets the number of '/' chars in the path
- * If there is no '/' then nothing changes since the path is same as deepestDirName
- * If there are '/'s then it sets the name of the folder as the deepestDirName
- * Makes a const name hyperLink put the path and deepestDirName inside the github hyperlink format
- * returns the hyperLink
+/** @function getHyperLinkFromPath  generates the clickable link to a specific folder
```

Comment:
Since this is the function the comment is on top of no need to repeat

Reply:
you mean I dont need 
...
* If there are '/'s then it sets the name of the folder as the deepestDirName
 * Makes a const name hyperLink put the path and deepestDirName inside the github hyperlink format
 * returns the hyperLink
?

Reply:
because I believe I have already deleted them

Reply:
Yes you don't need @function thoughjust type the description